examples/pyfunc/README.md

```diff
@@ -0,0 +1,42 @@
+# Pyfunc model examples
+
+This examples demonstrates the use of a pyfunc model with custom inference logic.
+More specifically:
+ - train a simple regression model
+ - create a *pyfunc* model that encapsulates the regression model with an attached module for custom inference logic
+
+# Structure of this example
+
+This examples contains a ``train.py`` file that trains a scikit-learn model with iris dataset and uses MLflow Tracking APIs to log the model.
+
+Then a nested **mlflow run** delivers the packaging of ``pyfunc`` model and ``custom_code`` module is attached
+to act as a custom inference logic layer in inference time.
+```
+├── custom_code
+│   ├── __init__.py
+│   └── custom_code.py
+├── train.py
+├── __init__.py
+```
+
+# Steps to reproduce
+
+1. Start an mlflow server
+```
+(terminal1) $ mlflow server  --default-artifact-root mlruns
```

Comment:
What is the purpose of this command?

Reply:
An up and running mlflow server instance is need in order to log and serve the models

Reply:
`mlflow models serve` should work without `mlflow server`.

Reply:
@sniafas Can we remove this command?

Reply:
@harupy if i run `mlflow models serve` in `examples/pyfunc` with existing logged runs under `mlruns/' , it does not serve the model and hults. An active `mlflow server` should be up in order to work
[img](https://imgur.com/a/x0oknKu)

Am I missing something ?

Reply:
Did you set the `MLFLOW_TRACKING_URI` variable?

Reply:
@sniafas Can you check the MLFLOW_TRACKING_URI is defined? If yes, remove it, log the model again, and then serve it. It should work.